src/components/pages/MyCases/MyCases.js

```diff
@@ -9,40 +9,19 @@ import Box from '@material-ui/core/Box';
 import { makeStyles } from '@material-ui/core/styles';
 import { DataGrid, GridColumnsToolbarButton } from '@material-ui/data-grid';
 import { Button, Typography } from 'antd';
+import { ReloadOutlined } from '@ant-design/icons';
 import './MyCases.less';
 export default function MyCases(props) {
   const { Title } = Typography;
   const [tabValue, setTabValue] = useState(0);
-  const { user } = props;
+  const { user, myPendingCases, getPendingCases } = props;
   const [myApprovedCases, setMyApprovedCases] = useState([]);
-  const [myPendingCases, setMyPendingCases] = useState([]);
   const [selectedTab, setSelectedTab] = useState(true);
 
   useEffect(() => {
-    trackPromise(
-      axios.get(
-        `${process.env.REACT_APP_API_URI}/pendingCases/:${user.userInfo.sub}`,
-        {
-          headers: {
-            Authorization: 'Bearer ' + user.authState.idToken.idToken,
-          },
-        }
-      )
-    )
-      .then(res => {
-        setMyPendingCases(
-          res.data.map(eachCase => {
-            return {
-              ...eachCase,
-              id: eachCase.case_number,
-            };
-          })
-        );
-      })
-      .catch(err => {
-        console.log(err);
-      });
-  }, [user.authState.idToken.idToken]);
+    getPendingCases();
+    // eslint-disable-next-line
+  }, []);
```

Comment:
Nice work moving the axios call to clean up this component.

Reply:
Agreed - loving the separation of concerns here!